### ISSUE TITLE
Adding handling of non-uniform tensors in DALI Pytorch plugin

### ISSUE THREAD

**Issue by JanuszL** [NVIDIA/DALI#2365]
It would be good to handle sparse tensors in the DALI Pytorch plugin, so it could output non-dense data that is not uniform across the batch, like bounding boxes or labels for the detection. GluonIterator SPARSE_TAG and DENSE_TAG could be a good reference for how the API and the behavior may look like.

**Comment by BogdanRuzh** [comment]
For those who desperately don't want to use Tensorflow's SparseTensor - just fn.pad your beautiful PyTorch Tensor

**Comment by JanuszL** [comment]
It is available now thanks to [Pytorch `DALIRaggedIterator`](https://docs.nvidia.com/deeplearning/dali/user-guide/docs/plugins/pytorch_plugin_api.html#nvidia.dali.plugin.pytorch.DALIRaggedIterator).
